config/backend/.env

```diff
@@ -0,0 +1,33 @@
+DOI_PREFIX="00.0000/"
+EXPRESS_SESSION_SECRET="<EXPRESS_SESSION_SECRET>"
+HTTP_MAX_REDIRECTS=5
+HTTP_TIMEOUT=5000
+JWT_SECRET=012334
+JWT_EXPIRES_IN=3600
+LDAP_URL="ldaps://ldap.server.invalid:636/"
+LDAP_BIND_DN="<USERNAME>@server.invalid"
+LDAP_BIND_CREDENTIALS="<PASSWORD>"
+LDAP_SEARCH_BASE="<SEARCH_BASE>"
+LDAP_SEARCH_FILTER="(LDAPUsername={{username}})"
+LOGBOOK_ENABLED="no"
+LOGBOOK_BASE_URL="http://scichat-loopback:3000/scichatapi"
+LOGBOOK_USERNAME="logbookReader"
+LOGBOOK_PASSWORD="<LOGBOOK_PASSWORD>"
+METADATA_KEYS_RETURN_LIMIT=100
+METADATA_PARENT_INSTANCES_RETURN_LIMIT=100
+MONGODB_URI="mongodb://mongodb:27017/scicat"
+OAI_PROVIDER_ROUTE="<OAI_PROVIDER_ROUTE>"
+PID_PREFIX="PID.SAMPLE.PREFIX/"
+PUBLIC_URL_PREFIX="https://doi.invalid/detail/"
+PORT=3000
+RABBITMQ_ENABLED="no"
+RABBITMQ_HOSTNAME="localhost"
+RABBITMQ_USERNAME="rabbitmq"
+RABBITMQ_PASSWORD="rabbitmq"
+REGISTER_DOI_URI="https://mds.test.datacite.org/doi"
+REGISTER_METADATA_URI="https://mds.test.datacite.org/metadata"
+SITE="SAMPLE_SITE"
+ADMIN_GROUPS=admin,ingestor,archivemanager
+CREATE_DATASET_GROUPS=group1,group2,group3

```

Comment:
shall we change this to match the groups in the seed data, rather than changing the seed values? I have the feeling this PR does too many things otherwise, as well as creating potential inconsistencies when rerunning the docker-compose up on an existing mongo_db

Reply:
Both need to be in sync ( this configures the BE which groups are allowed to create datasets, the  values in the seed data set the ownership  and are loaded bypassing  the backend)